### ISSUE TITLE
🐞 Unable to pan map or move markers in second window

### ISSUE THREAD

**Issue by ImmersiveWorks** [javalent/obsidian-leaflet#421]
### Check for existing bug reports before submitting.

- [X] I searched for existing Bug Reports and found no similar reports.

### Expected Behavior

Panning the map and adding pins/moving those pins should work when viewing the note containing the map in a new window (_Right Click > Open in New Window)_.

### Current behaviour

The map cannot be panned in the new window and pins cannot be moved. Additionally, if viewing the note in both the old and new window, the pin cannot be moved on the _old_ screen, either.

### Reproduction

1. Create a test map in a note:
```leaflet
id: Test
```

2. Open the note in a second window (_Right Click > Open in New Window)_.
3. Try to pan the map and/or move a pin added to the map.

### Which Operating Systems are you using?

- [ ] Android
- [ ] iPhone/iPad
- [ ] Linux
- [ ] macOS
- [X] Windows

### Obsidian Version Check

Current version: v1.4.11 (Installer version: v1.3.5)

### Plugin Version

Version 5.7.3

### Confirmation

- [X] I have disabled all other plugins and the issue still persists.

**Comment by Batres3** [comment]
Just want to add that this is the same as #371, which was closed without solution

**Comment by mrmaffen** [comment]
This is still an issue unfortunately. In my DnD campaign I regularly want to have my maps on a second screen. Therefore I have to open it in a new window and yea... This bug/issue can get very annoying.

Btw it's caused by this issue in leaflet.js https://github.com/Leaflet/Leaflet/issues/1368 and https://github.com/Leaflet/Leaflet/issues/2054

I wonder if obsidian-leaflet could alter the way it initializes the leaflet.js instance so as to avoid this reference issue

**Comment by Tamei** [comment]
Yes, same for me. I also want to use maps on a second screen and Leaflet is currently sadly not usable for this use case for me. I always have to open the map image files and all the nice Leaflet features are wasted.

**Comment by sunphoenixcmyk** [comment]
same in the latest version...It's easy to just open it with the shell associated application but it's to bad it doesn't have pins
